<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSettingsDialog</class>
 <widget class="QDialog" name="QSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings </string>
  </property>
  <layout class="QGridLayout" name="gridLayout_14">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>INI File</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QTableWidget" name="grdValues">
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="cboSections"/>
         </item>
         <item row="0" column="0">
          <widget class="QComboBox" name="cboINIFiles"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="cmdAddLine">
           <property name="text">
            <string>Add Key</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdDelete">
           <property name="text">
            <string>Delete Key</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>HTTP Server</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QCheckBox" name="swHttpEnabled">
            <property name="text">
             <string>enable HTTP Server</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>FTP Server</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QCheckBox" name="swFtpEnabled">
              <property name="text">
               <string>enable FTP Server on port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edtFTPPort"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="chkAnonymous">
            <property name="text">
             <string>Anonymous login enabled</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="lblFTPUser">
              <property name="text">
               <string>User:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edtFTPUser"/>
            </item>
            <item>
             <widget class="QLabel" name="lblFTPPassword">
              <property name="text">
               <string>Password:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edtFTPPassword"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Proxy</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string> Proxy enable</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string> Proxy user</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="edtProxyUser"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string> Proxy password</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="edtProxyPassword"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string> Proxy IP Address</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="edtProxyIpAddress"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string> Proxy Port</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="edtProxyIpPort"/>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="swProxy">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>6</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Style Sheets</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Font</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="lblFontName">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblFontSize">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdFontSelect">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../tosec.qrc">
                <normaloff>:/icon/toolbar/blue/images/toolbar/blue/setttings.ico</normaloff>:/icon/toolbar/blue/images/toolbar/blue/setttings.ico</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>89</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Available StyleSheets</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_4">
            <property name="text">
             <string>Fusion</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>Windows</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButton_5">
            <property name="text">
             <string>Fusion (black)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>WindowsXP</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton_6">
            <property name="text">
             <string>Bronze</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="radioButton_3">
            <property name="text">
             <string>Windows Vista</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="radioButton_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Norwegian Wood</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QRadioButton" name="radioButton_8">
            <property name="text">
             <string>none</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Available QSS files</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QComboBox" name="cboQssFiles"/>
            </item>
            <item>
             <widget class="QPushButton" name="cmdQssEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../tosec.qrc">
                <normaloff>:/icon/toolbar/blue/images/toolbar/blue/setttings.ico</normaloff>:/icon/toolbar/blue/images/toolbar/blue/setttings.ico</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="grpButtonColor">
         <property name="title">
          <string>Button Icon Color</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QComboBox" name="cboButtonColor"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDropbox">
      <attribute name="title">
       <string>Dropbox</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QGroupBox" name="grpDropbox">
         <property name="title">
          <string>Dropbox settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QLabel" name="lblDropboxKey">
            <property name="text">
             <string>Key:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="edtDropboxKey"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblDropboxSecret">
            <property name="text">
             <string>Secret:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="edtDropboxSecret"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblDropboxToken">
            <property name="text">
             <string>Token:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="edtDropboxToken"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblDropboxDir">
            <property name="text">
             <string>Directory:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="edtDropboxDir"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>162</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>418</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdOK">
       <property name="text">
        <string>Save Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="text">
        <string>Exit</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../tosec.qrc"/>
 </resources>
 <connections/>
</ui>
